src/kits/tracker/TitleView.cpp

```diff
@@ -45,6 +45,7 @@ All rights reserved.
 #include <PopUpMenu.h>
 #include <Window.h>
 
+#include <iostream>
```

Comment:
Unused?

Reply:
Yup. I removed this in the latest commit.